arbcompress/compress_wasm.go

```diff
@@ -14,6 +14,8 @@ func brotliCompress(inBuf []byte, outBuf []byte, level int, windowSize int) int6
 
 func brotliDecompress(inBuf []byte, outBuf []byte) int64
 
+func verifyNamespace()
```

Comment:
What is arbcompress? I have no idea about why this function is here

Reply:
I put it in a random place, when I clean this up I'll put it in its own arb-vid package